src/BlazorWebFormsComponents/BaseWebFormsComponent.cs

```diff
@@ -226,21 +226,27 @@ public ValueTask DisposeAsync()
 
 		public BaseWebFormsComponent ()
 		{
+			// Get Access to the ComponentBase fields we need to wrap every component in a CascadingValue

```

Comment:
To speed things up a little, the result of these reflection api calls can be cached in static readonly fields, so they are only performed once.

Reply:
True - I totally agree.